code/modules/events/disease_outbreak.dm

```diff
@@ -83,81 +127,250 @@
+		if(ADV_DISEASE_HARMFUL)
+			possible_symptoms += list(
+				/datum/symptom/disfiguration,
+				/datum/symptom/hallucigen,
+				/datum/symptom/polyvitiligo,
+				/datum/symptom/visionloss,
+			)
 
-/datum/event_admin_setup/disease_outbreak/advanced
-	///Admin selected custom severity rating for the event
-	var/max_severity
-	///Admin selected custom value for the maximum symptoms this virus should have
-	var/max_symptoms
+		if(ADV_DISEASE_DANGEROUS)
+			possible_symptoms += list(
+				/datum/symptom/alkali,
+				/datum/symptom/asphyxiation,
+				/datum/symptom/fire,
+				/datum/symptom/flesh_death,
+				/datum/symptom/flesh_eating,
+				/datum/symptom/narcolepsy,
+				/datum/symptom/voice_change,
+			)
 
-/datum/event_admin_setup/disease_outbreak/advanced/prompt_admins()
-	var/candidate_count = candidate_check()
-	if(!candidate_check())
-		tgui_alert(usr, "There are no candidates eligible to recieve a disease!", "Error")
-		return ADMIN_CANCEL_EVENT
-	tgui_alert(usr, "[candidate_count] candidates found!", "Disease Outbreak")
+	for(var/i in 1 to max_symptoms)
+		var/datum/symptom/chosen_symptom = pick_n_take(possible_symptoms)
+		if(chosen_symptom)
+			var/datum/symptom/new_symptom = new chosen_symptom
+			symptoms += new_symptom
 
-	if(tgui_alert(usr,"Customize your virus?", "Glorified Debug Tool", list("Yes", "No")) == "Yes")
-		max_severity = tgui_input_number(usr, "Select a custom severity for your virus!", "Plague Incorporation!", 3, 8)
-		max_symptoms = tgui_input_number(usr, "How many symptoms do you want your virus to have?", "A pox upon ye!", 3, 15)
+	if(prob(50))
+		symptoms += new /datum/symptom/viralevolution
 
-/datum/event_admin_setup/disease_outbreak/advanced/apply_to_event(datum/round_event/disease_outbreak/advanced/event)
-	event.max_severity = max_severity
-	event.max_symptoms = max_symptoms
+	if(prob(50))
+		symptoms += new /datum/symptom/youth
+
+	Refresh()
+
+	name = "Sample #[rand(1, 9999)]"
+
+/**
+ * Assign virus properties
+ *
+ * Now that we've picked our symptoms and severity, we determine the other stats
+ * (Stage Speed, Resistance, Transmissibility)
+ * The LOW/MID percentiles can be adjusted in the defines.
+ * If the virus is severity DANGEROUS we do not hide it from health scanners at event start.
+ * If the virus is airborne, also don't hide it.
+ */
+/datum/disease/advance/random/event/AssignProperties()
+	var/transmissibility = rand(1, 100)
+
+	addtimer(CALLBACK(src, PROC_REF(MakeVisible)), ((ADV_ANNOUNCE_DELAY * 2) - 10) SECONDS)
+
+	if(length(properties))
+		spreading_modifier = max(CEILING(0.4 * properties["transmittable"], 1), 1)
+		cure_chance = clamp(7.5 - (0.5 * properties["resistance"]), 5, 10) // Can be between 5 and 10
+		stage_prob = max(0.5 * properties["stage_rate"], 1)
+		SetSeverity(properties["severity"])
+		visibility_flags |= HIDDEN_SCANNER
+		switch(severity)
+			if("Dangerous")
+				visibility_flags &= ~HIDDEN_SCANNER
+
+			if("BIOHAZARD")
+				visibility_flags &= ~HIDDEN_SCANNER
```

Comment:
```suggestion
		switch(severity)
			if("Dangerous", "BIOHAZARD")
				visibility_flags &= ~HIDDEN_SCANNER
```

Should've told you that you could have just squashed it down this far by doing it like this.

Reply:
ahh.